sr_example/scripts/sr_example/sr_mix_examples_unsafe.py

```diff
@@ -10,6 +10,14 @@
 hand_commander = SrHandCommander()
 arm_commander = SrArmCommander()
 
+rospy.loginfo("Set arm teach mode ON")
+arm_commander.set_teach_mode(True)
+# sleep for some time during which the arm can be moved around by pushing it
+# but be careful to get away before the time runs out. You are warned
+rospy.sleep(20.0)
```

Comment:
OK I didn't understand it properly my bad. I thought it was the sleep for the time it took to switch the controllers.